app/assets/stylesheets/articles.scss

```diff
@@ -471,20 +208,20 @@
           height: 100%;
           width: 100%;
           &.round {
-            border-radius: 30px;
+            border-radius: 50%;
           }
         }
       }
       .content {
         float: left;
-        width: calc(100% - 75px);
-        padding: calc(1vw + 7px) 9px;
-        padding-bottom: 110px;
+        width: calc(100% - 72px);
+        box-sizing: border-box; // TODO: this should be applied globally, not only .content
+        padding: 16px 16px 100px 16px; // TODO: my, oh my...
         h3 {
           margin: 0px;
-          font-size: 21px;
-          line-height: 25px;
-          width: 100%;
+          font-size: 20px;
+          font-weight: 700;
```

Comment:
Should we also change the article title font weight to `700` in this PR? Even though this is primarily front-page-focused, we should ensure we keep other areas consistent eh?